README.md

```diff
@@ -86,6 +86,7 @@ To build the full software stack one needs to execute the following command in t
 cd catkin_ws
 
 catkin build
+# alternatively: catkin_make
```

Comment:
I would strongly discourage people from building with catkin_make. I would not mention it.

Reply:
I build with catkin_make only. But I see your point - it seems more robust to changes etc..